builders/manage/sudo/.pages

```diff
@@ -5,6 +5,7 @@ nav:
   - 'Manage the Sudo Account': 'sudo.md'
   - 'Dispatch Runtime Upgrades': 'upgrade.md'
   - 'Enabling Maintenance Mode': 'maintenance.md'
+  - 'Pausing Transactions': 'pause-transactions.md'
```

Comment:
can you please update these to be "Enable Maintenance Mode" and "Pause Transactions" so they are consistent with the rest of the titles in this section?